pkgs/development/python-modules/keyring-pass/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, gnupg
+, keyring
+, pass
+, poetry-core
+, pythonOlder
+}:
+let
+  pname = "keyring-pass";
+  version = "0.8.1";
+  module_name = builtins.replaceStrings [ "-" ] [ "_" ] pname;
+in
```

Comment:
```suggestion
```

Reply:
there is still a matter of `version` used in 2 places

Reply:
You can take care of that using `rec`.

Reply:
isn't `rec` kind of hard to follow antipattern?

Reply:
For Python packages there is no alternative yet.